elements/pfe-toast/README.md

```diff
@@ -0,0 +1,86 @@
+# PFElements Toast Element
+
+## Overview
+
+`pfe-toast` is a self-contained alert that is hidden on page load and slides in/out of the view when programmatically opened/closed.
+
+## Usage
+
+```html
+<pfe-toast>
+    <p>You've been successfully toasted!</p>
+</pfe-toast>
+```
+
+## Slots
+
+### Default slot
+The default slot can contain any type of content.
+
+## Attributes
+
+- `auto-dismiss`: Optionally dismiss the toast alert after the number of milliseconds provided. For example, `auto-dismiss="3000"` will dismiss the toast alert after three seconds.
```

Comment:
Would it be too complex to add support for `3s` or `100ms` for example?

Reply:
Sure! I can definitely add support for that.